test/togeojson-kml.test.js

```diff
@@ -90,12 +90,14 @@ test('[KML togeojson] fails empty features only', function(assert) {
   });
 });
 
-test('[KML togeojson] convert and index valid kml', function(assert) {
+test.only('[KML togeojson] convert, index, and archive valid kml', function(assert) {
```

Comment:
Thanks for catching this! I removed the `.only` and re-ran the tests which were 👍 .